src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java

```diff
@@ -308,37 +311,51 @@ private boolean hasAskedToNotReceiveFutureRequests( WorkflowDocument workflowDoc
 				if ( variableKey.startsWith( KewApiConstants.RECEIVE_FUTURE_REQUESTS_BRANCH_STATE_KEY )
 						&& variableValue.toUpperCase().equals( KewApiConstants.RECEIVE_FUTURE_REQUESTS_BRANCH_STATE_VALUE )
 						&& variableKey.contains( principalId ) ) {
+					LOG.info("hasAskedToReceiveFutureRequests: receiveFutureRequests = true");
 					receiveFutureRequests = true;
-					break;
 				}
 				else if ( variableKey.startsWith( KewApiConstants.RECEIVE_FUTURE_REQUESTS_BRANCH_STATE_KEY )
 						&& variableValue.toUpperCase().equals( KewApiConstants.DONT_RECEIVE_FUTURE_REQUESTS_BRANCH_STATE_VALUE )
 						&& variableKey.contains( principalId ) ) {
+					LOG.info("hasAskedToReceiveFutureRequests: doNotReceiveFutureRequests = true");
```

Comment:
Should this be LOG.debug instead? Any concerns about the number of log entries caused by the LOG.info?